Formula/hysteria.rb

```diff
@@ -20,7 +20,7 @@ class Hysteria < Formula
   depends_on "go" => :build
 
   def install
-    ldflags = "-s -w -X main.appVersion=v#{version} -X main.appDate=#{time.rfc3339} -X main.appCommit=#{Utils.git_short_head}"
+    ldflags = "-s -w -X main.appVersion=v#{version} -X main.appDate=#{time.strftime("%FT%T#{time.utc? ? 'Z' : '%:z'}")} -X main.appCommit=#{Utils.git_short_head}"
```

Comment:
```suggestion
    ldflags = %W[
      -s -w
      -X main.appVersion=v#{version}
      -X main.appDate=#{time.strftime("%FT%TZ")}
      -X main.appCommit=#{Utils.git_short_head}
    ]
```
Let's convert this to an array since we're already here.